clang/test/Preprocessor/opencl-macro-target-specific.cl

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 %s -E -dM -triple amdgcn-amdhsa-amdhsa \
+// RUN: | FileCheck --check-prefix=CHECK-AMDGPU %s
```

Comment:
Please add a comment here about what the test does.

Reply:
Comment added